src/internal/auth/interceptor.go

```diff
@@ -207,7 +208,7 @@ var authHandlers = map[string]authHandler{
 	"/pps.API/GarbageCollect":  authDisabledOr(authenticated),
 	"/pps.API/UpdateJobState":  authDisabledOr(authenticated),
 	"/pps.API/ListPipeline":    authDisabledOr(authenticated),
-	"/pps.API/ActivateAuth":    authDisabledOr(authenticated),
+	"/pps.API/ActivateAuth":    admin,
```

Comment:
It doesn't make sense to call `ActivateAuth` unless auth is enabled and the user is an admin, make this consistent between PFS and PPS.